### DISCUSSION TITLE
Need help in DNS setup

### DISCUSSION THREAD

**Discussion by electronic33** [vercel/community#444] [Help] [answered]
I have a NextJS frontend project on Vercel and a Node.js API on AWS (Elastic Beanstalk). I would like to have the webapp's (frontend) domain to be **example.com** and the API's (backend) to be **api.example.com**. How can I do this? I can't seem to be able to route from Vercel DNS to AWS Elastic Beanstalk.

Thanks in advance :)

**Comment by paulogdm** [comment, marked as answer]
Please take a look at https://vercel.com/knowledge/how-to-manage-vercel-dns-records. AWS should give you the records to use in the "api" subdomain.

**Reply by electronic33** [reply]
Could you elaborate more on the 'AWS should give you the records to use in the "api" subdomain' part please? I'm not very good at managing domain records, but I have no idea what you mean by that. 

Much thanks in advance :)

**Reply by paulogdm** [reply]
AWS should indicate if you need to add:

- A single CNAME record.
- A records.
- A CAA record (in rare cases).
- Etc.

**Reply by electronic33** [reply]
It doesn't :(

**Reply by j4nos** [reply]
Yes, would you elaborate plase. I would add a Heroku backend app. Docs is not clear for me either. https://vercel.com/support/articles/pointing-subdomains-to-external-services

**Reply by Pieparker** [reply]
The third party would be required to provide the record they need, in order for us to "map" to their service.

**Reply by jaankoppe** [reply]
I have an EC2 instance with Elastic IP, where can I find that "required record" ?

**Reply by Pieparker** [reply]
It's likely that you'll need to use the IP as an A record. However, AWS should provide instructions on communicating to an EC2 instance.

**Reply by paulogdm** [reply]
I am locking the discussion for now given how old it is. Please create a new one if needed 👍
